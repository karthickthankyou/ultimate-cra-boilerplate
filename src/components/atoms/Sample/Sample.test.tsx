import React from 'react'
import { render } from '@testing-library/react'
import Sample from './Sample'

describe('Sample Component', () => {
  test('it should match the snapshot', () => {
    const { asFragment } = render(<Sample />)
    expect(asFragment()).toMatchSnapshot()
  })
})
