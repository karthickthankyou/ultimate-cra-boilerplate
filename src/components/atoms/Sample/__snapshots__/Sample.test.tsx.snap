// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample Component it should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="p-4 border "
  >
    Some component!
  </div>
</DocumentFragment>
`;
